src/validate_txt.py

```diff
@@ -1,27 +1,23 @@
 import sys
 sys.path.append('./src')
+
 from validate_date_format import iso8601_format
 import pandas as pd
-import numpy as np
 
-# read text file into pandas DataFrame and
-# create header
 df = pd.read_csv("datelist.txt", sep=",",header=None ,on_bad_lines='skip',
                  names=["Order ID","Customer Name","Dates","Product ID","Quantity","Total Price"])
-df.index = np.arange(1, len(df) + 1)
-total_row = len(df)
-current_line = 0
+df.index = list(range(1, len(df) + 1))
+
 total_valid = 0
 total_invalid = 0
 list_invalid_dateformat = []
 list_valid_dateformat = []
-for i in range(total_row):
-    current_line += 1
-    if iso8601_format(df.loc[i+1,"Dates"]):
-        list_valid_dateformat.append(current_line)
+for row, column in df.iterrows():
```

Comment:
I should be
``` python
for index, row in df.iterrows():
 if iso8601_format(row["Dates"]):
 ...
```

Reply:
Okay Kubb but It's just names and has no effect to functions so , Should I use this one or replace it with the old one Kubb